src/Microsoft.ML.OnnxTransformer/OnnxTransform.cs

```diff
@@ -687,6 +723,422 @@ public NamedOnnxValue GetNamedOnnxValue()
+            /// <paramref name="isSrcColumn"/> is true), or to an "iinfo" index of an added column
+            /// (when <paramref name="isSrcColumn"/> is false).
+            /// </summary>
+            /// <param name="isSrcColumn">Whether the return index is for a source column</param>
+            /// <param name="col">The column index for this schema</param>
+            /// <returns>The index (either source index or iinfo index)</returns>
+            public int MapColumnIndex(out bool isSrcColumn, int col)
+            {
+                Contracts.Assert(0 <= col && col < _colMap.Length);
+                int index = _colMap[col];
+                if (index < 0)
+                {
+                    index = ~index;
+                    Contracts.Assert(index < AddedColumnIndices.Count);
+                    isSrcColumn = false;
+                }
+                else
+                {
+                    Contracts.Assert(index < InputSchema.Count);
+                    isSrcColumn = true;
+                }
+                return index;
+            }
+
+            /// <summary>
+            /// The given predicate maps from output column index to whether the column is active.
+            /// This builds an array of bools of length Input.ColumnCount containing the results of calling
+            /// predicate on the output column index corresponding to each input column index.
+            /// </summary>
+            public bool[] GetActiveInput(Func<int, bool> predicate)
+            {
+                Contracts.AssertValue(predicate);
+
+                var active = new bool[InputSchema.Count];
+                for (int dst = 0; dst < _colMap.Length; dst++)
+                {
+                    int src = _colMap[dst];
+                    Contracts.Assert(-AddedColumnIndices.Count <= src && src < InputSchema.Count);
+                    if (src >= 0 && predicate(dst))
+                        active[src] = true;
+                }
+                return active;
+            }
+        }
+
+        private class OnnxDataTransform : RowToRowTransformBase, IRowToRowMapper
+        {
+            // MYTODO: Is it even worth it to have this OnnxDataTransform class when it (including the RowImpl and Cursor)
+            // are identical to the RowToRowMapperTransform? The differences are:
+            // - This one expects specifically a OnnxTransformer.Mapper as _mapper from where to get the GetColumnsNames, whereas RTRMT expects a generic IRowMapper
+            // - This one has a _bindings object which is off type OnnxTransformer.Bindings, whereas RTRMT expects a generic ColumnsBindings
+            // - This one in here has a differend override for the Save method
+            // - This one in here doesn't have (but I don't know if it could have) methods related to SaveOnnx, SavePfa, ApplyToData, and VersionInfo of RTRMT.
+            // - RTRMT has an extra member called "_mapperFactory" that is used in ApplyToData
```

Comment:
The only differences between RowToRowMapperTransform and OnnxDataTransform are listed here. The main and central one is simply that OnnxDataTransform uses the Bindings class I added in this PR. Pretty much all the code is exactly the same.

So I don't know if it's worth it to copy all of this code. I can think of alternatives to enable ColumnBindings to drop columns, so that there's no need to create the Binding class in here and the OnnxDataTransform in here, and reuse the existing classes instead. Would have to test if it works.

So, what do people think? Is it ok to copy all of these code, or should I try to reuse the existing one in the other classes since it's nearly identical?